docs/source/credits.rst

```diff
@@ -16,7 +16,7 @@ We would like to thank the following sponsors for their financial support in the
 
 **Citation**
 
-Please use the following Bibtex citation when referencing this software. :cite:`sreehari2016plug` (coming soon) :cite:`venkatakrishnan2013plug`
+Please use the following bibtex citation when referencing this software. :cite:`sreehari2016plug` (coming soon) :cite:`venkatakrishnan2013plug`
```

Comment:
I assume that this line was copied from a previous project, and should be deleted.